ts/components/messagges/MessageDetailsComponent.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from "react";
```

Comment:
the dir name is `messagges`, need to be `messages`

Reply:
Fixed